frontend/src/components/limited-textarea/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+import { Text, Textarea, TextareaProps } from "theme-ui";
+
+type LimitedTextareaPros = TextareaProps & {
+  limit: number;
+};
+const LimitedTextarea = ({
+  value,
+  limit,
+  onChange,
+  ...props
+}: LimitedTextareaPros) => {
+  const [content, setContent] = useState(value.slice(0, limit));
+
+  return (
+    <>
+      <Textarea
```

Comment:
browsers have a `maxlength` property to automatically handle this, woudln't it work better than changing the content like this?